subscriptions/providers/apple_in_app/__init__.py

```diff
@@ -99,7 +97,7 @@ def check_payments(self, payments: Iterable[SubscriptionPayment]):
 
     @classmethod
     def _get_validated_in_app_product(cls, response: AppleVerifyReceiptResponse) -> AppleInApp:
-        if not response.is_valid or response.receipt.bundle_id != cls.bundle_id:
+        if not response.is_valid or response.receipt.bundle_id != settings.APPLE_BUNDLE_ID:
```

Comment:
Do not bake payment provider to `settings` module. Here your internal method depends on settings, while we want to maybe use settings for default init values.

Reply:
Going back to the one defined as the class var.